doc/api/crypto.md

```diff
@@ -1039,6 +1048,16 @@ the `signature_format` which can be `'latin1'`, `'hex'` or `'base64'`.
 If a `signature_format` is specified, the `signature` is expected to be a
 string; otherwise `signature` is expected to be a [`Buffer`][].
 
+The optional `options` argument is an object which specifies additional
```

Comment:
as for sign, `signature_format` should be possible to pass as `encoding` in the options

unrelated, but 

> If a `signature_format` is specified, the `signature` is expected to be a
  string; otherwise `signature` is expected to be a [`Buffer`][].

is a bizarre restriction, I wonder if it is even accurate? I will investigate.